tests/system/providers/google/cloud/cloud_run/example_cloud_run_service.py

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that uses Google Cloud Run Service Operators.
+"""
+
+from __future__ import annotations
+
+import os
+from datetime import datetime
+
+from google.cloud.run_v2 import Service
+from google.cloud.run_v2.types import k8s_min
+
+from airflow.models.dag import DAG
+from airflow.providers.google.cloud.operators.cloud_run import (
+    CloudRunCreateServiceOperator,
+)
+
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
+DAG_ID = "example_cloud_run_serivce"
+
+region = "us-central1"
+service_name_prefix = "cloudrun-system-test-service"
```

Comment:
Such names of objects should be provided with DAG_ID and ENV_ID (check other system tests for examples), to be sure that every name will be unique

Reply:
add ENV_ID and check that every name will be unique!